### DISCUSSION TITLE
plotly

### DISCUSSION THREAD

**Discussion by darubiano** [flet-dev/flet#4332] [Packages] [answered]
https://pypi.org/project/plotly/

**Comment by ndonkoHenri** [comment, marked as answer]
plotly is a pure-Python package - so should work normally on mobile.

However, not "everything" might work. For example, methods like `fig.write_image(...)` / `fig.to_image(...)` need `kaleido`, which is a Chrome/Node-based static-export binary. Kaleido has no mobile build and no recipe planned; in such cases, you'd need to, for example, render server-side and ship the image instead.

Give it a try and let know if you face issues.
